cmd/harmony.go

```diff
@@ -250,6 +255,11 @@ func main() {
 	currentNode.Role = node.NewNode
 
 	if *isBeacon {
+		var beacondb ethdb.Database
+		if *dbSupported {
+			beacondb, _ = InitLDBDatabase(*ip, *port, *freshDB, true)
```

Comment:
this is not correct. this logic will happen when it is not beaconchain

Reply:
oops, fixed